src/main/java/org/usfirst/frc/team4099/lib/util/Rotation2D.kt

```diff
@@ -1,121 +0,0 @@
-package org.usfirst.frc.team4099.lib.util
-
```

Comment:
why was this file deleted?

Reply:
I think I mistakenly added it but it may have been the wrong one